package.json

```diff
@@ -2,61 +2,79 @@
   "name": "bind-event-listener",
   "version": "3.0.0",
   "private": false,
-  "description": "Making binding and unbinding DOM events easier",
+  "description": "Safe DOM Event binding and unbinding",
   "author": "Alex Reardon <alexreardon@gmail.com>",
   "license": "MIT",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/alexreardon/bind-event-listener.git"
-  },
   "keywords": [
+    "event listener",
+    "event handler",
+    "dom events",
     "dom",
     "ui",
-    "event listener",
-    "event handler"
+    "bind"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/alexreardon/bind-event-listener.git"
+  },
   "sideEffects": false,
+  "type": "module",
+  "exports": {
+    "types": {
+      "import": "./dist/index.d.js",
+      "require": "./dist/index.d.cjs"
+    },
+    "import": "./dist/index.js",
+    "require": "./dist/index.cjs"
+  },
   "config": {
-    "prettier_target": "src/**/*.{ts,js,jsx,md,json} test/**/*.{ts,js,jsx,md,json}"
+    "prettier_target": "src/**/*.ts test/**/*.ts"
   },
   "files": [
     "/dist",
     "/src"
   ],
   "size-limit": [
     {
-      "path": "dist/index.js",
-      "limit": "549B"
+      "path": "./dist/bind.js",
+      "limit": "95B"
+    },
+    {
+      "path": "./dist/bind-all.js",
+      "limit": "211B"
     }
   ],
+  "engines": {
+    "node": ">=18"
+  },
   "scripts": {
-    "typescript:check": "tsc --noEmit && tsc --noEmit --project ./test/tsconfig.json",
-    "typescript:watch": "tsc --watch",
-    "typescript:build": "tsc",
-    "prettier:check": "prettier --debug-check $npm_package_config_prettier_target",
+    "build": "yarn build:clean && yarn build:dist",
+    "build:clean": "rimraf dist",
+    "build:dist": "tsup",
+    "check": "yarn check:typescript && yarn check:prettier && yarn check:dist",
+    "check:prettier": "prettier --debug-check $npm_package_config_prettier_target",
+    "check:typescript": "tsc --noEmit && tsc --noEmit --project ./test/tsconfig.json",
+    "check:dist": "attw --pack . --ignore-rules no-resolution",
```

Comment:
I had to add `--ignore-rules no-resolution` as I haven't added a `node10` mechanism. I could fix that by adding a "main" field that points to the CJS output - but I am not sure that I should

Reply:
I have added `"main"` for now - but not sure if I should keep it

Reply:
Sorry this PR has changed so much 🤭